webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -47,7 +48,7 @@ class BreadcrumbBar extends React.Component {
               totalPages={totalPages}
               onTogglerClick={() => toggleSwitcher()}
               onHide={() => closeSwitcher()}
-              onOpen={() => this.handleOpen()}
+              onOpen={() => handleOpen || this.handleOpen()}
```

Comment:
1. You don't actually run the `handleOpen` method!
    When clicking the toggle button in the storybook, I don't see an action-logger about it.

2. Do we actually need the extra prop `handleOpen`?
    `handleOpen` should be called after we already decieded to open the switcher but need the data inside it.
    Maybe it is better to use the `toggleSwitcher`, `closeSwitcher` and `loadSwitcherResourcesByResource` to demo this component?